cmd/kubeadm/app/cmd/upgrade/plan_test.go

```diff
@@ -447,9 +447,10 @@ _____________________________________________________________________
 			},
 			versionStates: versionStates,
 			expectedBytes: []byte(`Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
-COMPONENT   CURRENT       TARGET
-kubelet     1 x v1.19.2   v1.19.3
-            2 x v1.19.3   v1.19.3
+COMPONENT         CURRENT   TARGET
+kubelet (node1)   v1.19.2   v1.19.3
+kubelet (node2)   v1.19.3   v1.19.3
+kubelet (node3)   v1.19.3   v1.19.3
```

Comment:
If a cluster has 100 nodes, the table will be too long.

Reply:
i think that's OK and it's more verbose.
that's what users want normally right?